acquia_lift.module

```diff
@@ -39,3 +40,56 @@ function acquia_lift_page_attachments(array &$page) {
   // Attach lift CSS to all pages.
   $page['#attached']['library'][] = 'acquia_lift/acquia-lift';
 }
+
+/**
+ * Implements hook_toolbar().
+ */
+function acquia_lift_toolbar() {
+  $links = array();
+  // Add a link to enable Lift 3.
+  $links['login_to_acquia_lift'] = array(
+    'title' => t('Login'),
+    // This will make the href attribute empty.
+    'url' => Url::fromUserInput('#'),
```

Comment:
Using a `href="#"` URL is a bad practice in web dev. This is not linking anywhere. That's a sure sign you _should not use a link_.

This is triggering some JS UI. So, use a button instead. Look at `contextual_toolbar()` for an example of this in core (and `contextual.toolbar.js` is the associated JS).